src/settings/OrderApprovals/OrderApprovals.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import { ConfigManager } from '@folio/stripes/smart-components';
+
+import {
+  MODULE_ORDERS,
+} from '../../components/Utils/const';
+import { getOrderApprovalsSetting } from '../../common/utils/getOrderApprovalsSetting';
+import OrderApprovalsForm from './OrderApprovalsForm';
+import css from './OrderApprovals.css';
+
+class OrderApprovals extends Component {
```

Comment:
seems it should be functional component, but `props.stripes.connect`

Reply:
I think we can't change to stateless component, because then this form does't work.